### PR TITLE
Remove metalness workarounds

### PR DESCRIPTION
**Pull request by bsavery**
### PURPOSE
Latest RPR SDK includes a fix for Uber metalness mode.  Now when in metalness mode the "Reflection Weight" acts as the "Specular" input in Disney type shaders.  This removes the need for various workarounds in code.

### EFFECT OF CHANGE
Particularly, the effect is that "specular" input changes the facing angle reflection, not the edge reflection.  Closer matching Principled and other shaders.

### NOTES FOR REVIEWERS
I haven't checked transmissive materials (glass).  Requires latest sdk.

### CODE REVIEW

**Review by bnagirniak — APPROVED** [review]
Tested - works ok.

File: src/rprblender/nodes/blender_nodes.py [hunk 1]
```diff
@@ -767,24 +767,23 @@ def enabled(val: [NodeItem, None]):
...
 
-        if enabled(specular) or enabled(metallic):
-            # setting reflection weight as max of specular and metallic weights
-            rpr_node.set_input(pyrpr.MATERIAL_INPUT_UBER_REFLECTION_WEIGHT, specular.max(metallic))
-            rpr_node.set_input(pyrpr.MATERIAL_INPUT_UBER_REFLECTION_ROUGHNESS, roughness)
-            rpr_node.set_input(pyrpr.MATERIAL_INPUT_UBER_REFLECTION_IOR, ior)
+        # setting reflection weight as max of specular and metallic weights
+        rpr_node.set_input(pyrpr.MATERIAL_INPUT_UBER_REFLECTION_WEIGHT, specular)
+        rpr_node.set_input(pyrpr.MATERIAL_INPUT_UBER_REFLECTION_ROUGHNESS, roughness)
+        #rpr_node.set_input(pyrpr.MATERIAL_INPUT_UBER_REFLECTION_IOR, ior)
```
[the remark this answers is not in the document]
Comment (bsavery) [reply src/rprblender/nodes/blender_nodes.py:773]: No, metalness = 0 is a pure dielectric (still needs specular setting)
if transmission is enabled, then refraction weight is set.

[the remark this answers is not in the document]
Comment (ardraeiss) [reply src/rprblender/nodes/blender_nodes.py:773]: I meant the IOR value setting for reflection.

[the remark this answers is not in the document]
Comment (bsavery) [reply src/rprblender/nodes/blender_nodes.py:773]: IOR in metalness mode is derived from the metalness value.

**Review by ardraeiss — APPROVED** [review]
OK

**Review by ardraeiss — APPROVED** [review]
Code is fine

**Review by bnagirniak — APPROVED** [review]
Tested - works ok

**Pull request merged**